src/components/auth/Register.jsx

```diff
@@ -1,9 +1,332 @@
+            label="first-name"
+            value={firstName}
+            onChange={handleFirstNameChange}
+            placeholder=" "
+            className="basis-2/4">
+            First Name
+          </FormInput>
+          <FormInput
+            type="text"
+            id="last-name"
+            label="last-name"
+            value={lastName}
+            onChange={handleLastNameChange}
+            placeholder=" "
+            className="basis-2/4">
+            Last Name
+          </FormInput>
+        </div>
+        {userRole === 'student' && (
+          <>
+            <div className="flex w-full gap-x-4 items-end">
+              <div className="basis-2/4">
+                <FormInput
+                  type="date"
+                  id="dob"
+                  label="dob"
+                  value={dateOfBirth}
+                  onChange={handleDateOfBirthChange}
+                  placeholder=" "
+                >
+                  Date of Birth
+                </FormInput>
+              </div>
+              <div className="basis-2/4">
+                {adultNameError && <FormErrorMsg error={adultNameError} />}
+                <FormInput
+                  type="text"
+                  id="adult-name"
+                  label="adult-name"
+                  value={adultName}
+                  onChange={handleAdultNameChange}
+                  placeholder=" "
+                  disabled={!isAdultNameRequired}>
+                  Adult Full Name
+                </FormInput>
+              </div>
+            </div>
+          </>
+        )}
+        {emailError && <FormErrorMsg error={emailError} />}
+        <FormInput
+          type="email"
+          id="email"
+          label="email"
+          value={email}
+          onChange={handleEmailChange}
+          placeholder=" ">
+          Email Address
+        </FormInput>
+        {confirmEmailError && <FormErrorMsg error={confirmEmailError} />}
+        <FormInput
+          type="email"
+          id="email2"
+          label="email2"
+          value={confirmEmail}
+          onChange={handleConfirmEmailChange}
+          placeholder=" ">
+          Confirm Email Address
+        </FormInput>
+        {passwordError && <FormErrorMsg error={passwordError} />}
+        <FormInput
+          type="password"
+          id="password"
+          label="password"
+          value={password}
+          onChange={handlePasswordChange}
+          placeholder=" ">
+          Password
+        </FormInput>
+        {confirmPasswordError && <FormErrorMsg error={confirmPasswordError} />}
+        <FormInput
+          type="password"
+          id="password2"
+          label="password2"
+          value={confirmPassword}
+          onChange={handleConfirmPasswordChange}
+          placeholder=" ">
+          Confirm Password
+        </FormInput>
+        <FormSubmitBtn
+          value={`Create ${userRole} account`}
+        />
+      </form>
+
+      <Modal
```

Comment:
I really like this notification. But it's more user friendly to automatically log user in. No need to change this but something to keep in mind. People loathe having to consistently enter information :)

Reply:
I agree that it can be frustrating to have to enter the same information that you just entered twice during the registration process. However, our team has chosen to redirect users to the login page after registration as it is a common practice and easier to implement.

Reply:
Sounds good!